components/bingx/actions/market-interface/bingx-get-ticker.mjs

```diff
@@ -2,7 +2,7 @@ import bingx from "../../bingx.app.mjs";
 
 export default {
   name: "BingX Market GetTicker",
-  version: "0.0.1",
+  version: "0.0.5",
```

Comment:
Looks like the version is wrong

Reply:
@vellames  - corrected it now. Was restructuring code a bit. Also, yes, have deliberately put it in draft. There are 11 more actions - yet to be developed :)